vizro-ai/.readthedocs.yaml

```diff
@@ -11,6 +11,6 @@ build:
     python: "3.11"
   commands:
     - pip install hatch
-    - cd vizro-ai/ && hatch run docs:mkdocs build --strict
```

Comment:
It's OK to remove --strict here because you've added it to the `toml` file, right?